packages/cfd/src/Containers/cfd-compare-accounts/cfd-instruments-label-highlighted.tsx

```diff
@@ -1,20 +1,45 @@
 import React from 'react';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
 import InstrumentsIconWithLabel from './instruments-icon-with-label';
 import { TInstrumentsIcon, TCompareAccountsCard } from 'Components/props.types';
 import { getHighlightedIconLabel } from '../../Helpers/compare-accounts-config';
 import { useStore } from '@deriv/stores';
 
 const CFDInstrumentsLabelHighlighted = ({ trading_platforms }: TCompareAccountsCard) => {
     const { traders_hub } = useStore();
-    const selected_region = traders_hub.selected_region;
+    const { selected_region, is_eu_user } = traders_hub;
 
     const iconData: TInstrumentsIcon[] = [...getHighlightedIconLabel(trading_platforms, selected_region)];
 
     return (
         <div className={'compare-cfd-account-outline'} data-testid='dt_compare_cfd_account_outline__container'>
-            {iconData.map(item => (
-                <InstrumentsIconWithLabel key={item.text} {...item} className='compare-cfd-account-instrument-icon' />
-            ))}
+            {is_eu_user ? (
+                <React.Fragment>
+                    {iconData
+                        .filter(item => item.highlighted)
+                        .map(item => (
+                            <InstrumentsIconWithLabel
+                                key={item.text}
+                                {...item}
+                                className='compare-cfd-account-instrument-icon'
+                            />
+                        ))}
+                    <div className='compare-cfd-account-card-container__eu-clients'>
+                        <Text color='red' size='xxs' weight='bold'>
+                            <Localize i18n_default_text='*Boom 300 and Crash 300 Index' />
```

Comment:
I think we have removed a testcase related to this. Is this content valid?

Reply:
yes, corresponding test case added to packages/cfd/src/Containers/cfd-compare-accounts/__tests__/cfd-instruments-label-highlighted.spec.tsx